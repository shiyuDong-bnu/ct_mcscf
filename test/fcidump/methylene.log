2025-05-10:10:01:23,432 INFO     [psi4.driver.task_planner:285] PLANNING Atomic:  keywords={'D_CONVERGENCE': 1e-08, 'E_CONVERGENCE': 1e-10, 'MAXITER': 300, 'REFERENCE': 'RHF', 'SCF_TYPE': 'PK', 'forte__scf_type': 'PK', 'forte__e_convergence': 1e-10, 'forte__d_convergence': 1e-08, 'forte__reference': 'RHF', 'forte__maxiter': 300, 'forte__basis': 'CC-PVDZ', 'function_kwargs': {}}
2025-05-10:10:01:23,434 INFO     [psi4.driver.driver:523] Compute energy(): method=forte, basis=cc-pvdz, molecule=default, nre=6.014699644935847
2025-05-10:10:01:23,966 INFO     [psi4.driver.driver:526] Return energy(): -38.940822086935434
