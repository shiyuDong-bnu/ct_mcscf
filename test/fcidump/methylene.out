
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.9.1 release

                         Git: Rev {HEAD} f53cdd7 


    D. G. A. Smith, L. A. Burns, A. C. Simmonett, R. M. Parrish,
    M. C. Schieber, R. Galvelis, P. Kraus, H. Kruse, R. Di Remigio,
    A. Alenaizan, A. M. James, S. Lehtola, J. P. Misiewicz, M. Scheurer,
    R. A. Shaw, J. B. Schriber, Y. Xie, Z. L. Glick, D. A. Sirianni,
    J. S. O'Brien, J. M. Waldrop, A. Kumar, E. G. Hohenstein,
    B. P. Pritchard, B. R. Brooks, H. F. Schaefer III, A. Yu. Sokolov,
    K. Patkowski, A. E. DePrince III, U. Bozkaya, R. A. King,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, C. D. Sherrill,
    J. Chem. Phys. 152(18) 184108 (2020). https://doi.org/10.1063/5.0006002

                            Additional Code Authors
    E. T. Seidl, C. L. Janssen, E. F. Valeev, M. L. Leininger,
    J. F. Gonthier, R. M. Richard, H. R. McAlexander, M. Saitow, X. Wang,
    P. Verma, M. H. Lechner, A. Jiang, S. Behnle, A. G. Heide,
    M. F. Herbst, and D. L. Poole

             Previous Authors, Complete List of Code Contributors,
                       and Citations for Specific Modules
    https://github.com/psi4/psi4/blob/master/codemeta.json
    https://github.com/psi4/psi4/graphs/contributors
    http://psicode.org/psi4manual/master/introduction.html#citing-psifour

    -----------------------------------------------------------------------


    Psi4 started on: Saturday, 10 May 2025 10:01AM

    Process ID: 3140501
    Host:       debianlab
    PSIDATADIR: /home/sydong/install_tutorial/psi4/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    Molecule: geometry: Molecule is not complete, please use 'update_geometry'
                    once all variables are set.
Molecule: Setting geometry variable r_ch to 2.102300
Molecule: Setting geometry variable ahch to 101.710000

Scratch directory: /home/sydong/scratch/psi4/

  Forte
  ----------------------------------------------------------------------------
  A suite of quantum chemistry methods for strongly correlated electrons

    git branch: ct_forte - git commit: 41562031

  Developed by:
    Francesco A. Evangelista, Chenyang Li, Kevin P. Hannon,
    Jeffrey B. Schriber, Tianyuan Zhang, Chenxi Cai,
    Nan He, Nicholas Stair, Shuhe Wang, Renke Huang
  ----------------------------------------------------------------------------

  Forte will use options passed as a dictionary. Option read from psi4 will be ignored


  Preparing forte objects from a Psi4 Wavefunction object Using basis set CC-PVDZ from Psi4 global options.   => Libint2 <=

    Primary   basis highest AM E, G, H:  6, 6, 3
    Auxiliary basis highest AM E, G, H:  7, 7, 4
    Onebody   basis highest AM E, G, H:  -, -, -
    Solid Harmonics ordering:            Gaussian

*** tstart() called on debianlab
*** at Sat May 10 10:01:23 2025

   => Loading Basis Set <=

    Name: CC-PVDZ
    Role: ORBITAL
    Keyword: BASIS
    atoms 1   entry C          line   138 file /home/sydong/install_tutorial/psi4/share/psi4/basis/cc-pvdz.gbs 
    atoms 2-3 entry H          line    22 file /home/sydong/install_tutorial/psi4/share/psi4/basis/cc-pvdz.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              RHF Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C2v

    Geometry (in Bohr), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         C            0.000000000000     0.000000000000    -0.190863170104    12.000000000000
         H            0.000000000000    -1.630440522826     1.136287534046     1.007825032230
         H            0.000000000000     1.630440522826     1.136287534046     1.007825032230

  Running in c1 symmetry.

  Rotational constants: A =     19.80481  B =     11.23490  C =      7.16840 [cm^-1]
  Rotational constants: A = 593733.17106  B = 336813.79877  C = 214903.20347 [MHz]
  Nuclear repulsion =    6.014699644935847

  Charge       = 0
  Multiplicity = 1
  Electrons    = 8
  Nalpha       = 4
  Nbeta        = 4

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-10
  Density threshold  = 1.00e-08
  Integral threshold = 1.00e-12

  ==> Primary Basis <==

  Basis Set: CC-PVDZ
    Blend: CC-PVDZ
    Number of shells: 12
    Number of basis functions: 24
    Number of Cartesian functions: 25
    Spherical Harmonics?: true
    Max angular momentum: 2

  ==> Integral Setup <==

  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   3
      Number of AO shells:              12
      Number of primitives:             32
      Number of atomic orbitals:        25
      Number of basis functions:        24

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 90300 doubles for integral storage.
  We computed 3081 shell quartets total.
  Whereas there are 3081 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:              375
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1

  Minimum eigenvalue in the overlap matrix is 2.8712604800E-02.
  Reciprocal condition number of the overlap matrix is 7.7320335120E-03.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF (no occupation information).

   -------------------------
    Irrep   Nso     Nmo    
   -------------------------
     A         24      24 
   -------------------------
    Total      24      24
   -------------------------

  ==> Iterations <==

                        Total Energy        Delta E     RMS |[F,P]|

   @RHF iter SAD:   -38.17046734184489   -3.81705e+01   0.00000e+00 
   @RHF iter   1:   -38.87001134666033   -6.99544e-01   4.87804e-03 DIIS/ADIIS
   @RHF iter   2:   -38.88050821298012   -1.04969e-02   1.03345e-03 DIIS/ADIIS
   @RHF iter   3:   -38.88099703297717   -4.88820e-04   2.71486e-04 DIIS/ADIIS
   @RHF iter   4:   -38.88104336531818   -4.63323e-05   5.73986e-05 DIIS
   @RHF iter   5:   -38.88104507147467   -1.70616e-06   8.85278e-06 DIIS
   @RHF iter   6:   -38.88104514242077   -7.09461e-08   1.57360e-06 DIIS
   @RHF iter   7:   -38.88104514552549   -3.10472e-09   3.39040e-07 DIIS
   @RHF iter   8:   -38.88104514567171   -1.46223e-10   5.27250e-08 DIIS
   @RHF iter   9:   -38.88104514567461   -2.89191e-12   5.31933e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1A    -11.294184     2A     -0.887998     3A     -0.557433  
       4A     -0.390728  

    Virtual:                                                              

       5A      0.062788     6A      0.195244     7A      0.260123  
       8A      0.586212     9A      0.694948    10A      0.738118  
      11A      0.782650    12A      0.862641    13A      1.008432  
      14A      1.201842    15A      1.214557    16A      1.288106  
      17A      1.746015    18A      1.762820    19A      1.933018  
      20A      2.006488    21A      2.024872    22A      2.308185  
      23A      2.649152    24A      2.743767  

    Final Occupation by Irrep:
              A 
    DOCC [     4 ]
    NA   [     4 ]
    NB   [     4 ]

  @RHF Final Energy:   -38.88104514567461

   => Energetics <=

    Nuclear Repulsion Energy =              6.0146996449358472
    One-Electron Energy =                 -63.5308050345083046
    Two-Electron Energy =                  18.6350602438978505
    Total Energy =                        -38.8810451456746051

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix


 Multipole Moments:

 ------------------------------------------------------------------------------------
     Multipole            Electronic (a.u.)      Nuclear  (a.u.)        Total (a.u.)
 ------------------------------------------------------------------------------------

 L = 1.  Multiply by 2.5417464519 to convert [e a0] to [Debye]
 Dipole X            :         -0.0000000            0.0000000           -0.0000000
 Dipole Y            :          0.0000000            0.0000000            0.0000000
 Dipole Z            :         -0.3596734            1.1273960            0.7677226
 Magnitude           :                                                    0.7677226

 ------------------------------------------------------------------------------------

*** tstop() called on debianlab at Sat May 10 10:01:23 2025
Module time:
	user time   =       0.13 seconds =       0.00 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =       0.13 seconds =       0.00 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes


  ==> MO Space Information <==

  -------------------------------
                        A   Sum
  -------------------------------
    FROZEN_DOCC         0     0
    RESTRICTED_DOCC     1     1
    GAS1                6     6
    GAS2                0     0
    GAS3                0     0
    GAS4                0     0
    GAS5                0     0
    GAS6                0     0
    RESTRICTED_UOCC    17    17
    FROZEN_UOCC         0     0
    Total              24    24
  -------------------------------   => Loading Basis Set <=

    Name: STO-3G
    Role: ORBITAL
    Keyword: MINAO_BASIS
    atoms 1   entry C          line    61 file /home/sydong/install_tutorial/psi4/share/psi4/basis/sto-3g.gbs 
    atoms 2-3 entry H          line    19 file /home/sydong/install_tutorial/psi4/share/psi4/basis/sto-3g.gbs 


  State Singlet (Ms = 0) A GAS min: 0 0 0 0 0 0 ; GAS max: 12 0 0 0 0 0 ; weights:
      1.000000000000
  Forte will use psi4 integrals

  ==> Primary Basis Set Summary <==

  Basis Set: CC-PVDZ
    Blend: CC-PVDZ
    Number of shells: 12
    Number of basis functions: 24
    Number of Cartesian functions: 25
    Spherical Harmonics?: true
    Max angular momentum: 2


  JK created using conventional PK integrals
  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   3
      Number of AO shells:              12
      Number of primitives:             32
      Number of atomic orbitals:        25
      Number of basis functions:        24

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 90300 doubles for integral storage.
  We computed 3081 shell quartets total.
  Whereas there are 3081 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:              400
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1



  ==> Integral Transformation <==

  Number of molecular orbitals:                         24
  Number of correlated molecular orbitals:              24
  Number of frozen occupied orbitals:                    0
  Number of frozen unoccupied orbitals:                  0
  Two-electron integral type:                 Conventional


  Skip integral allocation and transformation for AO-driven CASSCF.

          -----------------------------------------------------------
                  Multi-Configurational Self Consistent Field
                Two-Step Approximate Second-Order AO Algorithm
            written by Chenyang Li, Kevin P. Hannon, and Shuhe Wang
          -----------------------------------------------------------


  ==> MCSCF Calculation Information <==

    --------------------------------------------------------
    Print level                                      Default
    Integral type                               CONVENTIONAL
    CI solver type                                       FCI
    Final orbital type                             CANONICAL
    Derivative type                                     NONE
    Optimize orbitals                                   TRUE
    Include internal rotations                         FALSE
    Debug printing                                     FALSE
    Energy convergence                             1.000e-08
    Gradient convergence                           1.000e-07
    Max value for rotation                         2.000e-01
    Max number of macro iter.                            100
    Max number of micro iter. for orbitals                 6
    Max number of micro iter. for CI                      12
    DIIS start                                            15
    Min DIIS vectors                                       3
    Max DIIS vectors                                       8
    Frequency of DIIS extrapolation                        1
    --------------------------------------------------------

  ==> Independent Orbital Rotations <==

    ORBITAL SPACES                         A
    ----------------------------------------
             ACTIVE / RESTRICTED_DOCC      6
    RESTRICTED_UOCC /          ACTIVE    102
    RESTRICTED_UOCC / RESTRICTED_DOCC     17
    ----------------------------------------

  ==> String Lists <==

    --------------------------------------------------------
    number of alpha electrons                              3
    number of beta electrons                               3
    number of alpha strings                               20
    number of beta strings                                20
    --------------------------------------------------------

  ==> FCI Solver <==

    --------------------------------------------------------
    Spin adapt                                         FALSE
    Number of determinants                               400
    Symmetry                                               0
    Multiplicity                                           1
    Number of roots                                        1
    Target root                                            0
    --------------------------------------------------------
  Initial guess space is incomplete.
  Adding 1 determinant(s).

  ==> Initial Guess <==

  Initial guess determinants:         51

  Classification of the initial guess solutions

  Number   2S+1   Selected
  ------------------------
    25       1       *
    24       3        
     2       5        
  ------------------------

    Spin    Root           Energy        <S^2>    Status
  -------------------------------------------------------
   singlet    0      -38.888213520897  -0.000000  added
  -------------------------------------------------------

  ==> Davidson-Liu Solver <==

    --------------------------------------------------------
    Print level                                      Default
    Energy convergence threshold                   1.000e-08
    Residual convergence threshold                 1.000e-06
    Schmidt orthogonality threshold                1.000e-12
    Schmidt discard threshold                      1.000e-07
    Size of the space                                    400
    Number of roots                                        1
    Maximum number of iterations                         100
    Collapse subspace size                                 2
    Maximum subspace size                                 10
    --------------------------------------------------------

  Davidson-Liu solver: adding 1 guess vectors
  Iteration     Average Energy            max(∆E)            max(Residual)  Vectors
  ---------------------------------------------------------------------------------
       0      -38.888213520897       38.888213520897        0.106795512989      1
       1      -38.896585900641        0.008372379744        0.021054158326      2
       2      -38.896991785291        0.000405884651        0.004699642779      3
       3      -38.897009777327        0.000017992036        0.001230007883      4
       4      -38.897011125635        0.000001348308        0.000235276085      5
       5      -38.897011167050        0.000000041415        0.000047167969      6
       6      -38.897011168630        0.000000001580        0.000007293585      7
       7      -38.897011168667        0.000000000037        0.000001278593      8
       8      -38.897011168668        0.000000000001        0.000000234627      9
  ---------------------------------------------------------------------------------

  ==> Root No. 0 <==

    222000     -0.98737221
    220200      0.12398417
    022200      0.04917571
    202002      0.03653395
    b2a200     -0.03308867
    a2b200     -0.03308867
    220002      0.02068082
    ab20ba     -0.01854315
    ba20ab     -0.01854315
    022020      0.01826012
    202200      0.01499556
    2ba0ba     -0.01490048
    2ab0ab     -0.01490048
    202020      0.01478476
    b2a020      0.01154879
    a2b020      0.01154879
    220020      0.01124798
    aa20bb      0.01046969
    bb20aa      0.01046969

    Total Energy:     -38.897011168668, <S^2>: 0.000000
    Time for FCI:       0.011424438000

  ==> Energy Summary <==

    Multi.(2ms)  Irrep.  No.               Energy      <S^2>
    --------------------------------------------------------
       1  (  0)     A     0      -38.897011168668   0.000000
    --------------------------------------------------------
  Computing RDMs for dipole moments ... Done

  ==> Summary of Dipole Moments [e a0] (Nuclear + Electronic) <==

       State           DM_X           DM_Y           DM_Z           |DM|
    --------------------------------------------------------------------
          0A    -0.00000000    -0.00000000     0.69304331     0.69304331
    --------------------------------------------------------------------
     Nuclear     0.00000000     0.00000000     1.12739605     1.12739605
    --------------------------------------------------------------------

  ==> Natural Orbitals <==

        2A      1.994507      3A      1.994252      4A      1.961230  
        5A      0.040682      6A      0.005598      7A      0.003732  

 sydong:  unsymmetrize 2RDM


  ==> MCSCF Iterations <==

                      Energy CI                    Energy Orbital
           ------------------------------  ------------------------------
    Iter.        Total Energy       Delta        Total Energy       Delta  Orb. Grad.  Micro
    ----------------------------------------------------------------------------------------
 sydong:  unsymmetrize 2RDM

       1     -38.897011168668 -3.8897e+01    -38.926158365521 -3.8926e+01  5.6849e-03    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       2     -38.939598627396 -4.2587e-02    -38.940595358000 -1.4437e-02  4.4558e-04    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       3     -38.940778685753 -1.1801e-03    -38.940810823057 -2.1547e-04  1.3483e-05    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       4     -38.940818832826 -4.0147e-05    -38.940821076549 -1.0253e-05  2.4401e-06    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       5     -38.940821763693 -2.9309e-06    -38.940821981508 -9.0496e-07  2.6944e-07    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       6     -38.940822052165 -2.8847e-07    -38.940822075407 -9.3899e-08  1.3253e-07    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       7     -38.940822083110 -3.0945e-08    -38.940822085684 -1.0277e-08  5.7722e-08    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       8     -38.940822086546 -3.4363e-09    -38.940822086837 -1.1534e-09  2.0884e-08    5/Y
    ----------------------------------------------------------------------------------------

  A miracle has come to pass: MCSCF iterations have converged!

  Performing final CI Calculation using converged orbitals

  ==> String Lists <==

    --------------------------------------------------------
    number of alpha electrons                              3
    number of beta electrons                               3
    number of alpha strings                               20
    number of beta strings                                20
    --------------------------------------------------------

  ==> FCI Solver <==

    --------------------------------------------------------
    Spin adapt                                         FALSE
    Number of determinants                               400
    Symmetry                                               0
    Multiplicity                                           1
    Number of roots                                        1
    Target root                                            0
    --------------------------------------------------------

  ==> Davidson-Liu Solver <==

    --------------------------------------------------------
    Print level                                      Default
    Energy convergence threshold                   1.000e-08
    Residual convergence threshold                 1.000e-06
    Schmidt orthogonality threshold                1.000e-12
    Schmidt discard threshold                      1.000e-07
    Size of the space                                    400
    Number of roots                                        1
    Maximum number of iterations                         100
    Collapse subspace size                                 2
    Maximum subspace size                                 10
    --------------------------------------------------------

  Davidson-Liu solver: restarting from previous calculation.
  Iteration     Average Energy            max(∆E)            max(Residual)  Vectors
  ---------------------------------------------------------------------------------
       0      -38.940822086837        0.000000000291        0.000011098654      1
       1      -38.940822086926        0.000000000090        0.000004038190      2
       2      -38.940822086935        0.000000000008        0.000001106934      3
       3      -38.940822086935        0.000000000001        0.000000252170      4
  ---------------------------------------------------------------------------------

  ==> Root No. 0 <==

    222000     -0.96748826
    220200      0.17822872
    202002      0.07204389
    a2b200     -0.06318349
    b2a200     -0.06318349
    022200      0.04471840
    022020      0.04229733
    ba20ab     -0.03897847
    ab20ba     -0.03897847
    202020      0.03761822
    2ab0ab     -0.03340618
    2ba0ba     -0.03340618
    220002      0.03117472
    a2b020      0.03047960
    b2a020      0.03047960
    ba20ba     -0.02893806
    ab20ab     -0.02893806
    2ab0ba     -0.02435973
    2ba0ab     -0.02435973
    220020      0.02375548
    022002      0.02158695
    200202     -0.01364551
    020220     -0.01281369
    22a0b0      0.01219480
    22b0a0      0.01219480
    bb20aa      0.01004040
    aa20bb      0.01004040

    Total Energy:     -38.940822086935, <S^2>: 0.000000
    Time for FCI:       0.005905573000

  ==> Energy Summary <==

    Multi.(2ms)  Irrep.  No.               Energy      <S^2>
    --------------------------------------------------------
       1  (  0)     A     0      -38.940822086935   0.000000
    --------------------------------------------------------
  Computing RDMs for dipole moments ... Done

  ==> Summary of Dipole Moments [e a0] (Nuclear + Electronic) <==

       State           DM_X           DM_Y           DM_Z           |DM|
    --------------------------------------------------------------------
          0A    -0.00000000     0.00000000     0.61571822     0.61571822
    --------------------------------------------------------------------
     Nuclear     0.00000000     0.00000000     1.12739605     1.12739605
    --------------------------------------------------------------------

  ==> Natural Orbitals <==

        2A      1.981905      3A      1.976889      4A      1.911821  
        5A      0.085505      6A      0.023015      7A      0.020865  

  Canonicalizing final MCSCF orbitals

  ==> Semicanonicalize Orbitals <==

    MIX INACTIVE ORBITALS         TRUE
    MIX GAS ACTIVE ORBITALS       FALSE
    GAS1                          CANONICAL
    INACTIVE_DOCC                 CANONICAL
    INACTIVE_UOCC                 CANONICAL

    Off-Diag. Elements       Max           2-Norm
    ------------------------------------------------
    GAS1                 0.0106252953   0.0169559982
    INACTIVE_DOCC        0.0000000000   0.0000000000
    INACTIVE_UOCC        0.1873218142   0.5064288238
    ------------------------------------------------

    Canonicalization test failed

  Timing for orbital canonicalization:                        0.000 s.

  !!! WARNING: Final Active Orbitals Maybe Different from the Original !!!

    Based on projections of maximum overlap method:
           6A -> 7A        RESTRICTED_UOCC
  Orbital rotations on spin-free RDMs ...
    Transformed 1 RDMs.
    Transformed 2 RDMs.

  Time to prepare integrals:        0.224 seconds
  Time to run job          :        0.306 seconds
  Total                    :        0.530 seconds
    Molecular point group: c1
    Full point group: C2v

    Geometry (in Bohr), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         C            0.000000000000     0.000000000000    -0.190863170104    12.000000000000
         H            0.000000000000    -1.630440522826     1.136287534046     1.007825032230
         H            0.000000000000     1.630440522826     1.136287534046     1.007825032230

  -AO BASIS SET INFORMATION:
    Name                   = CC-PVDZ
    Blend                  = CC-PVDZ
    Total number of shells = 12
    Number of primitives   = 32
    Number of AO           = 25
    Number of SO           = 24
    Maximum AM             = 2
    Spherical Harmonics    = TRUE

  -Contraction Scheme:
    Atom   Type   All Primitives // Shells:
   ------ ------ --------------------------
       1     C     17s 4p 1d // 3s 2p 1d 
       2     H     4s 1p // 2s 1p 
       3     H     4s 1p // 2s 1p 

  ==> AO Basis Functions <==

    [ CC-PVDZ ]
    spherical
    ****
    C   1
    S   8 1.00
                      6665.00000000           0.00069200
                      1000.00000000           0.00532900
                       228.00000000           0.02707700
                        64.71000000           0.10171800
                        21.06000000           0.27474000
                         7.49500000           0.44856400
                         2.79700000           0.28507400
                         0.52150000           0.01520400
    S   8 1.00
                      6665.00000000          -0.00014600
                      1000.00000000          -0.00115400
                       228.00000000          -0.00572500
                        64.71000000          -0.02331200
                        21.06000000          -0.06395500
                         7.49500000          -0.14998100
                         2.79700000          -0.12726200
                         0.52150000           0.54452900
    S   1 1.00
                         0.15960000           1.00000000
    P   3 1.00
                         9.43900000           0.03810900
                         2.00200000           0.20948000
                         0.54560000           0.50855700
    P   1 1.00
                         0.15170000           1.00000000
    D   1 1.00
                         0.55000000           1.00000000
    ****
    H   2
    S   3 1.00
                        13.01000000           0.01968500
                         1.96200000           0.13797700
                         0.44460000           0.47814800
    S   1 1.00
                         0.12200000           1.00000000
    P   1 1.00
                         0.72700000           1.00000000
    ****
    H   3
    S   3 1.00
                        13.01000000           0.01968500
                         1.96200000           0.13797700
                         0.44460000           0.47814800
    S   1 1.00
                         0.12200000           1.00000000
    P   1 1.00
                         0.72700000           1.00000000
    ****

